docs/modules/record-encryption/hashicorp-vault/con-vault-setup.adoc

```diff
@@ -72,12 +70,13 @@ capabilities = ["read", "write"]
 }
 ----
 
-== Filter Actor
+== Filter Token
```

Comment:
```suggestion
== Token authentication
```
Again I'm not really sure what `Filter Token` means. If you wanted a common structure between the documents maybe `Filter authentication: [IAM Identity|Token]` ?